portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,10 +3,20 @@
 public class Comment {
   private final String name;
   private final String comment;
+  private final long timestamp;
```

Comment:
Is it seconds? milliseconds? from 1970? why not java.time.Instant?

Reply:
I shall change it to timestampMillis, and add an annotation that it is using unix time. I was not aware of java.time.instant, but my only hesitation is with how this would be represented in datastore, and if converting in/out of this format to pass to frontend or datastore may cause inconsistencies in data.